frontend/components/Card.vue

```diff
@@ -0,0 +1,31 @@
+<template>
+    <div class="card">
+        {{ content }}
+    </div>
+</template>
+
+<script>
+    export default {
+        name: 'Card',
+        props: {
+            content: String
+        },
+        data () {
+            return {}
+        },
+    }
+</script>
+
+<style lang="scss" scoped>
+
+.card {
+    display: inline-block;
+    background-color: white;
+    padding: 20px;
+    color: var(--dark-color);
+    border: 2px solid var(--dark-color);
+    border-radius: var(--border-radius);
+    box-shadow: var(--shadow-length) var(--shadow-length) var(--dark-color);
+}
+
+</style>
```

Comment:
nit: missing newline at end of file
```suggestion
</style>

```